### DISCUSSION TITLE
HTML output from the translation .json file

### DISCUSSION THREAD

**Discussion by NitrousBGC** [i18next/next-i18next#1899] [Q&A] [answered]
Hello, 

I've recently started to use **next-i18next** with **Next JS** and i'm having some trouble with finding a way to output html from the .json file. 
Is it possible ? Or do I need to make a new key for every strong, br or span element ?

Here is what I mean 

![next-i18](https://user-images.githubusercontent.com/108579761/177640120-afc13a0b-d456-434d-85b8-a9a495dc7826.png)

Thanks for your help !

**Comment by adrai** [comment, marked as answer]
Have you already tried to use the Trans component? https://react.i18next.com/latest/trans-component

**Reply by NitrousBGC** [reply]
Hello! I forgot to post an update about it but I found out about the Trans component a couple hours ago. It did the job perfectly, I can output br, strong, and i without any problem directly from the .json file. 

Thanks for your help!
